doc/devdocs/modules/launcher/new-plugin-checklist.md

```diff
@@ -0,0 +1,30 @@
+# New plugin checklist
+- A plugin is a project under `modules\launcher\Plugins`
```

Comment:
```suggestion
- [ ] The plugin is a project under `modules\launcher\Plugins`
```
Let's add a checkbox to all entries, so it can be easily copied and pasted in new plugin PRs for signing off.
And instead of `A plugin..` let's use `The plugin`